src/Http/Controllers/Corporation/LedgerController.php

```diff
@@ -97,4 +97,114 @@ public function getPlanetaryInteractionByMonth(int $corporation_id, $year = null
             compact('pidates', 'pitotals', 'piimport', 'piexport',
                 'corporation_id', 'month', 'year'));
     }
+    /**
+     * @param      $corporation_id
+     * @param null $year
+     * @param null $month
+     *
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     */
+    public function getOfficeRentalFeeByMonth(int $corporation_id, $year = null, $month = null)
+    {
+
+        ! is_null($year) ? $year : $year = date('Y');
+        ! is_null($month) ? $year : $month = date('m');
+
+        $officerentalfeedates = $this->getCorporationLedgerOfficeRentalFeeDates($corporation_id);
+
+        $officerentalfeetotals = $this->getCorporationLedgerOfficeRentalFeeTotalsByMonth(
+            $corporation_id, $year, $month);
+
+        return view('web::corporation.ledger.officerentalfee',
+            compact('officerentalfeedates', 'officerentalfeetotals',
+                'corporation_id', 'month', 'year'));
+    }
+    /**
+     * @param      $corporation_id
+     * @param null $year
+     * @param null $month
+     *
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     */
+    public function getIndustryFacilityTaxByMonth(int $corporation_id, $year = null, $month = null)
+    {
+
+        ! is_null($year) ? $year : $year = date('Y');
+        ! is_null($month) ? $year : $month = date('m');
+
+        $industryfacilitytaxdates = $this->getCorporationLedgerIndustryFacilityTaxDates($corporation_id);
+
+        $industryfacilitytaxtotals = $this->getCorporationLedgerIndustryFacilityTaxTotalsByMonth(
+            $corporation_id, $year, $month);
+
+        return view('web::corporation.ledger.industryfacilitytax',
```

Comment:
`industryfacilitytax` is not an existing blade. This call must be `web::corporation.ledger.industryfacilitytaxbymonth`